### ISSUE TITLE
How to use the Docker?

### ISSUE THREAD

**Issue by tal987** [lovato/react-native-android#1]
Hi, is the this docker working? and its working for react native 0.60.5?
can you explain how to use it?

Thank you for your work

**Comment by lovato** [comment]
No, it is not. I tried to make it work in the past, and eventually, I will see if it worth trying again, since new tools arose which made all setup different.
